content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,273 @@
+* When the `_type` parameter is not provided and the token includes the Provenance read scope, all relevant Provenance resources associated with each of the non-provenance resources will be exported by default. You MAY use the  `includeAssociatedData` parameter to return only the latest provenance.
+* When the `_type` parameter is provided with specific resources and includes 'Provenance', all relevant Provenance resources associated with each of the non-provenance resources will be exported by default.
+* When both the `includeAssociatedData` parameter and the `_type` parameter are provided, the `_type` parameter MUST include `Provenance`
+
+### Headers
+
+<%= headers head: {'Accept': 'application/fhir+json', 'Prefer': 'respond-async', Authorization: '&lt;OAuth2 Bearer Token>'} %>
+
+Optional Header
+
+<%= headers head: {'Prefer': 'handling=lenient'} %>
+
+When the Prefer: handling=lenient header is provided any unknown or unsupported parameters will be ignored as specified [here](http://hl7.org/fhir/uv/bulkdata/STU2/export.html#query-parameters).
+
+### Example
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Group/11ec-d16a-c763b73e-98e8-a31715e6a2bf/$export
+
+Response
+
+    Status: 202 Accepted
+    Content-Location: https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Status Request
+
+Checks the status of the export job.
+Use the URL returned in the `Content-Location` header from the kick-off request to retrieve the status of a bulk data export:
+
+    GET /bulk-export/jobs/<job_id>
+
+_Notes_
+
+* Clients should follow an Exponential Backoff approach when polling for status as outlined in the [bulk data export specification.](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#bulk-data-status-request)
+
+### Authorization Types
+
+<%= authorization_types(provider: false, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {'Accept': 'application/fhir+json', Authorization: '&lt;OAuth2 Bearer Token>'} %>
+
+### Example
+
+#### In-Progress Response
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+Response
+
+    Status: 202 Accepted
+    X-Progress: in-progress
+    Retry-After: 600
+
+#### Complete Response
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+Response
+
+    Status: 200 OK
+    Expires: Mon, 22 Jul 2025 23:59:59 GMT
+    Content-Type: application/json
+
+<%= json(:R4_BULK_EXPORT_COMPLETE) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## File Retrieval Request
+
+Once an export is complete, the status endpoint will return file URLs which may be used to download the exported resources
+
+    GET /bulk-export/files/<file_id>
```

Comment:
same here lets make it generic like below
`GET <File URL from status response>`

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/eff91ed66335ec7340cb5a2ec8a51663c1bd9cbf#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR195